tests/publish_subscribe.rs

```diff
@@ -5,9 +5,13 @@ use up_rust::{MockUListener, UMessageBuilder, UTransport, UUri};
 
 mod common;
 
-#[tokio::test(flavor = "multi_thread")]
-#[ignore = "should only be executed with an MQTT broker running"]
+#[tokio::test]
 async fn test_publish_and_subscribe() {
+    env_logger::builder().parse_filters("debug").init();
+
+    // fixture
+    let (_mosquitto, broker_port) = common::start_mosquitto().await;
```

Comment:
Will this container be stopped again at a later time?

Reply:
Stop is implemented with Drop in the crate